charlatan/tests/test_testcase.py

```diff
@@ -50,7 +50,7 @@ def test_install_all_fixtures(self):
         self.uninstall_all_fixtures()
 
         fixtures = self.install_all_fixtures()
-        self.assertEqual(len(fixtures), 3)
+        self.assertEqual(len(fixtures), 4)
```

Comment:
This unit test (`TestTestCase`) uses a fixture_manager which loads `relationships_without_models.yaml` that now contains 4 fixtures instead of 3.